hosts/azure/jenkins-controller/jenkins-casc.yaml

```diff
@@ -164,3 +164,163 @@ tool:
   mavenGlobalConfig:
     globalSettingsProvider: "standard"
     settingsProvider: "standard"
+# https://github.com/jenkinsci/job-dsl-plugin
+# https://jenkinsci.github.io/job-dsl-plugin
+jobs:
+  - script: >
```

Comment:
That's a lot of duplication in here. Can we keep this generated by Nix, and call a helper function with the different arguments for things that differ?